src/framework/components/element/image-element.js

```diff
@@ -989,6 +989,15 @@ class ImageElement {
         }
 
         this._material = value;
+
+        // Remove material asset if changed
+        if (this._materialAsset) {
+            const asset = this._system.app.assets.get(this._materialAsset);
+            if (!asset || asset.resource !== value) {
+                this.materialAsset = null;
```

Comment:
Should this line be `this._materialAsset = null`? as calling the setter for `materialAsset` also sets the `_material` to null as well?